src/scvmm/azext_scvmm/_params.py

```diff
@@ -0,0 +1,265 @@
+# pylint: disable=too-many-statements
+
+from knack.arguments import CLIArgumentType
+from azure.cli.core import AzCommandsLoader
+from .vendored_sdks.models import AllocationMethod
+from .scvmm_constants import BusType, VHDType
+from ._actions import VmNicAddAction, VmDiskAddAction
+
+
+def load_arguments(self: AzCommandsLoader, _):
+    from azure.cli.core.commands.parameters import (
+        tags_type,
+        get_enum_type,
+        get_three_state_flag,
+    )
+    from azure.cli.core.commands.validators import (
+        get_default_location_from_resource_group,
+    )
+
+    scvmm_name_type = CLIArgumentType(
+        options_list=['--name', '-n'],
+        help='Name of the resource.',
+        id_part='name',
+    )
+
+    custom_location_type = CLIArgumentType(
+        options_list=['--custom-location'],
+        help='Name or ID of the custom location that will manage this resource.',
+    )
+
+    vmmserver_type = CLIArgumentType(
+        options_list=['--vmmserver', '-v'],
+        help='Name or ID of the vmmserver that is managing this resource.',
+    )
+
+    uuid_type = CLIArgumentType(
+        options_list=['--uuid'],
+        help="The ID of the resource created in the VMM.",
+    )
+
+    inventory_item_type = CLIArgumentType(
+        options_list=['--inventory-item', '-i'],
+        help='Name or ID of the inventory item.',
+    )
+
+    force_delete_type = CLIArgumentType(
+        options_list=['--force'],
+        help='Force the resource to be deleted from azure.',
+        action='store_true',
+    )
+
+    with self.argument_context('scvmm') as c:
+        c.argument('tags', tags_type)
+        c.argument('location', validator=get_default_location_from_resource_group)
+        c.argument('resource_name', arg_type=scvmm_name_type)
+        c.argument('custom_location', arg_type=custom_location_type)
+        c.argument('vmmserver', arg_type=vmmserver_type)
+        c.argument('uuid', arg_type=uuid_type)
+        c.argument('inventory_item', arg_type=inventory_item_type)
+        c.argument('force', arg_type=force_delete_type)
+
+    with self.argument_context('scvmm vmmserver connect') as c:
+        c.argument(
+            'fqdn', options_list=['--fqdn'], help="FQDN/IP address of the vmmserver."
+        )
+        c.argument(
+            'port', type=int, options_list=['--port'], help="The port of the vmmserver."
+        )
+        c.argument(
+            'username',
+            options_list=['--username'],
+            help="Username to use for connecting to the vmmserver.",
+        )
+        c.argument(
+            'password',
+            options_list=['--password'],
+            help="Username password credentials to use for connecting to the vmmserver.",
+        )
+
+    self.argument_context('scvmm cloud create')
+
+    self.argument_context('scvmm virtual-network create')
+
+    self.argument_context('scvmm vm-template create')
```

Comment:
what these lines used for?

Reply:
They were just placeholders for the list of commands. Have removed these lines since they are unused.